streams-designer/README.md

```diff
@@ -0,0 +1,155 @@
+# Simple load from Cloudant to Db2 in Streams Designer (Beta)
+This tutorial is an introduction on loading Cloudant documents and saving that data into Db2 within Streams Designer.
+In this tutorial you will:
+
+1. Set up IBM services Cloudant, Streaming Analytics, Watson Studio, and Db2 Warehouse on Cloud.
```

Comment:
maybe a `:` after IBM services, since it is a bit of a list

Reply:
Added in 38e8498.